### DISCUSSION TITLE
Simulating physically plausible solid-gas interactions

### DISCUSSION THREAD

**Discussion by oskars-t** [InteractiveComputerGraphics/SPlisHSPlasH#286] [Q&A]
Hi there. I'm working on a 3D simulation in which air is blown from a grid of emitters underneath a rigid object (e.g. a cube) to levitate it around. To what extent can SPlisHSPlasH be used for a realistic depiction of such an interaction? The focus here is that the emitted fluids and the surrounding space behave like air.

Thanks a lot in advance!

**Comment by janbender** [comment]
I think in this case you would have to sample to whole space using SPH particles so that the emitted air can interact with the air in the domain around the cube.
